libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -51,15 +50,20 @@ extern const AP_HAL::HAL& hal;
 #define AP_FOLLOW_OFFSET_TYPE_NED       0   // offsets are in north-east-down frame
 #define AP_FOLLOW_OFFSET_TYPE_RELATIVE  1   // offsets are relative to lead vehicle's heading
 
-#define AP_FOLLOW_ALTITUDE_TYPE_RELATIVE  1 // relative altitude is used by default   
+#define AP_FOLLOW_ALTITUDE_TYPE_ABSOLUTE  0 // altitudes are absolute
+#define AP_FOLLOW_ALTITUDE_TYPE_RELATIVE  1 // home relative altitude is used by default
+#define AP_FOLLOW_ALTITUDE_TYPE_ORIGIN    2 // origin relative altitude
+#define AP_FOLLOW_ALTITUDE_TYPE_TERRAIN   3 // terrain relative altitude
```

Comment:
I don't think these are used.

Reply:
Sorry yes - left over from before I switch to enum